From: Rafael J. Wysocki <rjw@suse.de>
Subject: Documentation: Fix typo in freezer-subsystem.txt
Patch-mainline: v3.2-rc2
Git-commit: 5f11161675be1351cadb96c83850e20bf5bcdc48
References: bnc#724544

Fix a typo in a PID in Documentation/cgroups/freezer-subsystem.txt.

Signed-off-by: Rafael J. Wysocki <rjw@suse.de>
---
 Documentation/cgroups/freezer-subsystem.txt |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

Index: linux-3.0-SLE11-SP2/Documentation/cgroups/freezer-subsystem.txt
===================================================================
--- linux-3.0-SLE11-SP2.orig/Documentation/cgroups/freezer-subsystem.txt
+++ linux-3.0-SLE11-SP2/Documentation/cgroups/freezer-subsystem.txt
@@ -33,9 +33,9 @@ demonstrate this problem using nested ba
 
 	From a second, unrelated bash shell:
 	$ kill -SIGSTOP 16690
-	$ kill -SIGCONT 16990
+	$ kill -SIGCONT 16690
 
-	<at this point 16990 exits and causes 16644 to exit too>
+	<at this point 16690 exits and causes 16644 to exit too>
 
 This happens because bash can observe both signals and choose how it
 responds to them.
